test/e2e/specs/colorsApiPage.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { vpTest } from '../fixtures/vpTest';
+import { expect, test } from '@playwright/test';
+import { waitForPageToLoadWithTimeout } from '../src/helpers/waitForPageToLoadWithTimeout';
+import { getLinkByName } from '../testData/pageLinksData';
+import { ExampleLinkName } from '../testData/ExampleLinkNames';
+
+const link = getLinkByName(ExampleLinkName.ColorsAPI);
+
+vpTest(`Test if 3 videos on colors API page are playing as expected`, async ({ page, pomPages }) => {
+    await test.step('Navigate to colors API page by clicking on link', async () => {
+        await pomPages.mainPage.clickLinkByName(link.name);
+        await waitForPageToLoadWithTimeout(page, 5000);
+    });
+    await test.step('Validating that the first video is playing (in case isPause is false)', async () => {
+        expect(await pomPages.colorsApiPage.colorsApiColorSkinVideoComponent.validateVideoPaused(false));
+    });
+    await test.step('Validating that the second video is playing (in case isPause is false)', async () => {
+        expect(await pomPages.colorsApiPage.colorsApiDarkSkinVideoComponent.validateVideoPaused(false));
+    });
+    await test.step('Scroll until the third video element is visible', async () => {
+        await pomPages.colorsApiPage.colorsApiLightSkinVideoComponent.locator.scrollIntoViewIfNeeded();
+    });
```

Comment:
I would name the videos by their configuration description so instead of first video i would say for example `validating that the modified colors video` or for example `the dark skin video` ... this will make it easier to understand what failed when tests fail.

Reply:
Modified the test steps description